src/features/commands/watch.js

```diff
@@ -0,0 +1,38 @@
+const { SlashCommandBuilder } = require('@discordjs/builders'),
+      db = require('../../db');
+
+module.exports = {
+    type: 'slash',
+    data: new SlashCommandBuilder()
+        .setName('watch')
+        .setDescription('Sends a direct message whenever keyword that you track gets mentioned.')
+        .addStringOption(option =>
+            option.setName('keyword')
+                .setDescription('The keyword you track.')
+                .setRequired(true)
+        ),
+    async execute (client, interaction) {
+        const keyword = interaction.options.getString("keyword");
+        const server = interaction.guild;
+        let watchedKeywordsCollection = db.getWatchedKeywords();
+        const refreshWatchedCollection = () => (
+             watchedKeywordsCollection = db.getWatchedKeywords()
+        );
+        if (keyword.length < 3) {
+          interaction.reply({ content: `Keyword ${keyword} cannot be watched because the keyword has fewer than 3 characters.`, ephemeral: true });
+        } else {
```

Comment:
As per the suggestion above, this else is not needed because if keyword.length < 3, the function is, in its suggested code, returned and therefore any code after it will not execute.

Reply:
It doesn't look like this was addressed, why is that @itsWindows11 ?

Reply:
Resolved it.